docs/Reference/EA/overview.md

```diff
@@ -49,7 +49,7 @@ The RPC-Go local configuration option does not communicate with a remote server
 [Configure EA using RPC-Go](./RPCConfiguration/rpcgoConfiguration.md){: .md-button .md-button--primary }
 
 <figure class="figure-image">
-    <img src="..\..\..\assets\images\EA_RPC_Architecture.png" width=75% alt="Figure 2: Enterprise Assistant Architecture using RPC-Go">
+    <img src="..\..\..\assets\images\diagrams\EA_RPC_Architecture.png" width=75% alt="Figure 2: Enterprise Assistant Architecture using RPC-Go">
```

Comment:
The EA images also need to be updated, as they still reference the `Open AMT Cloud Toolkit Component`.